frontend/views/search/_results.html.erb

```diff
@@ -5,20 +5,33 @@
 <%
   # Start of modifications
   # See https://github.com/archivesspace/archivesspace/blob/master/frontend/app/helpers/search_helper.rb
+  # Customize columns in search results if accessions might be included
+
   model = "accession"
-  add_pref_columns(model)
-  # Add content_description column at end
-  prop = 'content_description'
-  enum_locale_key = "#{model}_#{prop}"
-  add_column(
-    I18n.t("#{model}.#{prop}"),
-    { locale_key: enum_locale_key, model: model, sortable: false },
-    proc { |record|
-      v = record[prop] || ASUtils.json_parse(record['json'])[prop]
-      I18n.t("enumerations.#{enum_locale_key}.#{v}", :default => v.to_s) unless v.nil?
-    }
-  )
-  add_actions_column if !params[:linker] || params[:linker] === 'false'
+  if !@search_data.nil? & [model, "multi"].include?(@search_data.get_type)
+    # Add standard columns for results
+    add_columns
```

Comment:
Hidden inside of this call is logic that will add the multi-select needed by the linked entity browse page if there is a linker param. See https://github.com/archivesspace/archivesspace/blob/305b9a3a11579f28472e0a4948803c2a7a8e959d/frontend/app/helpers/search_helper.rb#L308